addons/mail/static/src/models/rtc_controller/rtc_controller.js

```diff
@@ -51,6 +51,9 @@ function factory(dependencies) {
          * @param {MouseEvent} ev
          */
         async onClickRejectCall(ev) {
+            if (this.callViewer.threadView.thread.hasPendingRtcRequest) {
+                return;
```

Comment:
I'm thinking we should sent a notification popup in all of those cases, because it looks like the current user interaction will be completely ignored for no visible reason to him.

I wonder how often that happens, but maybe we should have a queue instead of a complete ignore?

Do we really need to avoid doing these RPC if there is a pending request?

Reply:
@seb-odoo Currently, buttons are disabled when `hasPendingRtcRequest` is true so there is a functional feedback, so I suppose we could remove the check.

Reply:
The problem is that the render is async so it's still possible to be able to click on the button for a time even if it's already disabled in models.